packages/flutter/test/material/banner_test.dart

```diff
@@ -9,67 +9,117 @@ import 'package:flutter_test/flutter_test.dart';
 void main() {
   testWidgets('Custom background color respected', (WidgetTester tester) async {
     const Color color = Colors.pink;
-    await tester.pumpWidget(
-      MaterialApp(
-        home: MaterialBanner(
-          backgroundColor: color,
-          content: const Text('I am a banner'),
-          actions: <Widget>[
-            TextButton(
-              child: const Text('Action'),
-              onPressed: () { },
-            ),
-          ],
+    const String contentText = 'Content';
```

Comment:
For these old tests that are being changed, since we are maintaining backwards compatibility for static banners, we should keep these pre-existing tests and add new ones that are testing the functionality you have added.